keras_lr_finder/lr_finder.py

```diff
@@ -62,6 +62,29 @@ def find(self, x_train, y_train, start_lr, end_lr, batch_size=64, epochs=1):
         # Restore the original learning rate
         K.set_value(self.model.optimizer.lr, original_lr)
 
+    def find_generator(self, generator, start_lr, end_lr, batch_size=64, epochs=1):
```

Comment:
A lot of code in this function is duplicate. It would be nice to extract common code from find and find_generator functions.

Reply:
Yeah it's either through `len` or `steps_per_epoch`. New changes support both situations.